templates/commands/render/render.go

```diff
@@ -697,3 +755,16 @@ func destOK(fs fs.StatFS, dest string) error {
 
 	return nil
 }
+
+func runGitCmds(argsList [][]string, workingDir string) error {
```

Comment:
Can we use common.Run() for this? https://github.com/abcxyz/abc/blob/main/templates/common/exec.go#L44

Reply:
Updating...